src/Hooks.php

```diff
@@ -33,7 +31,7 @@ public function gutes_array_fields() {
 
 	public function get_block_data( $post ) {
 		if ( ! $this->API ) {
-			$this->API = API::init();
+			$this->API = new API();
```

Comment:
This leads to multiple API objects, instead eliminate this entire check, and change the constructor